### DISCUSSION TITLE
"Exception succesfully sended to server" event

### DISCUSSION THREAD

**Discussion by sednik** [getsentry/sentry-dotnet#1237] [Q&A]
Hi, 
i'm trying to understand if there's a way to know if a notification is correctly sended to the Sentry cloud.
The reason is to show to the user the feedback.

i'm using this instruction:
`SentryId id = SentrySdk.CaptureException(ex);`
and i don't see any exception or notification fired if the notification is sended or not (eg: no internet connection) to the cloud.

**Comment by bruno-garcia** [comment]
Even if the envelope is sent over the network, there's no guarantee the event won't be dropped due to some server-side filter, quota limit or [something else](https://forum.sentry.io/t/sentry-responds-http-200-with-event-id-but-no-event-on-dashboard/10888). That said, this does come up a lot. Including in the [JS SDK](https://github.com/getsentry/sentry-javascript/issues/2939), [Dart SDK](https://github.com/getsentry/sentry-dart/issues/352). Currently the only way to get a 'best effort' callback is to add a custom (decorator) HTTP Message handler: https://github.com/getsentry/sentry-dotnet/blob/4d36f002d7ab641d25e260b98a1966dee3d7dcbd/src/Sentry/SentryOptions.cs#L341 that fires after the underlying handler completes a request.
